pkg/handler/commit.go

```diff
@@ -1,9 +1,24 @@
 package handler
 
 import (
+	"encoding/json"
+	"fmt"
+	"io"
 	"net/http"
+
+	"github.com/go-resty/resty/v2"
+	"github.com/gorilla/mux"
 )
 
+type commit struct {
+	SHA    string `json:"sha"`
+	Commit struct {
+		Author struct {
+			Date string `json:"date"`
+		} `json:"author"`
+	} `json:"commit"`
+}
```

Comment:
We are keeping the type definitions in a separate file in this project, it would be better to move this definition to commit_request_response